### PR TITLE
docs: Add license strategy doc

### LINKED ISSUE #58: Create licensing strategy document
We need a brief document explaining that the project uses SPDX license identifiers.

### PR DESCRIPTION
**Pull request by jodh-intel**
Add a brief document explaining that the project uses SPDX license
identifiers.

Fixes #58.

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

### CODE REVIEW

**Comment by jodh-intel** [comment]
@egernst, @klynnrif, @grahamwhaley - ptal.

**Comment by grahamwhaley** [comment]
lgtm.
I wonder if we should explicitly call out that the default license for the project is Apache2.0 ?

**Comment by jodh-intel** [comment]
Agreed and done.

File: Licensing-strategy.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+project is [Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0).
+
+## License file
+
+All repositories in the project have a top level file called `LICENSE`. This
+file lists full details of all licences used by the repository.
+
+## License for individual files
+
+Where possible all files in all repositories also contain an
```
Comment (klynnrif) [inline Licensing-strategy.md]: Changing "and" to "a": Where possible all files in all repositories also contain a

Comment (jodh-intel) [reply Licensing-strategy.md]: Ah, you mean "an" -> "a" ;) Done.

**Comment by jodh-intel** [comment]
Thanks @klynnrif.

@grahamwhaley - could you take a squizz?

**Comment by jodh-intel** [comment]
Ping @egernst, @WeiZhang555, @bergwolf.

**Pull request merged**